cmd/influx_tsm/main.go

```diff
@@ -135,7 +149,7 @@ func main() {
 	w.Init(os.Stdout, 0, 8, 1, '\t', 0)
 	fmt.Fprintln(w, "Database\tRetention\tPath\tEngine\tSize")
 	for _, si := range shards {
-		fmt.Fprintf(w, "%s\t%s\t%s\t%s\t%d\n", si.Database, si.RetentionPolicy, si.FullPath(dataPath), si.FormatAsString(), si.Size)
+		fmt.Fprintf(w, "%v\t%v\t%v\t%v\t%d\n", si.Database, si.RetentionPolicy, si.FullPath(opts.DataPath), si.FormatAsString(), si.Size)
```

Comment:
I'm still in the process of replacing all of the old `fmt.Fprintf(os.Stderr, ...); os.Exit(1)` with `log.Fatalf(...)`